.github/workflows/build.yml

```diff
@@ -35,6 +35,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         ghc: ["8.6.5", "8.8.4", "8.10.2"]
+        jdk: ["8", "15"]
```

Comment:
Curiously, I think you left a comment on an older version of this PR, as I reverted this change in the latest version (the rationale being that since JDK 9+ support is experimental, it's probably not worth increasing the CI runtimes to test them yet).

I agree that in an ideal world, instead of duplicating jobs we'd just test multiple versions of the JDK per GHC version. I decided to punt that problem for now, given the experimental status of JDK 9+ support.

Reply:
I've opened #1072 to track the task of making it easier to test multiple versions of the JDK without needlessly bloating CI times.